portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Stanley's Image Gallery</title>
+    <link href="https://fonts.googleapis.com/css2?family=Caveat&family=Montserrat&family=Nunito+Sans&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    
+  </head>
+  <body class="gallery">
+    <div class="content-wrap">
+        <h1>Nikita's Gallery</h1>
+        <h2>Here are some fun pictures of my life</h2>
+        <div class="row">
+            <div class="column">
+                <a href="images/Nikita1.JPG"><img src="images/Nikita1.JPG"/></a>
+                <a href="images/Nikita2.JPG"><img src="images/Nikita2.JPG"/></a>
+                <a href="images/Nikita3.JPG"><img src="images/Nikita3.JPG"/></a>
+                <a href="images/Nikita4.JPG"><img src="images/Nikita4.JPG"/></a>
+            </div>
+            <div class="column">
+                <a href="images/Nikita5.JPG"><img src="images/Nikita5.JPG"/></a>
+                <a href="images/Nikita6.jpg"><img src="images/Nikita6.jpg"/></a>
+                <a href="images/Nikita7.jpg"><img src="images/Nikita7.jpg"/></a>
+                <a href="images/Nikita8.jpg"><img src="images/Nikita8.jpg"/></a>
+            </div>
+            <div class="column">  
+                <a href="images/Nikita9.JPG"><img src="images/Nikita9.JPG"/></a>
+                <a href="images/Nikita10.JPG"><img src="images/Nikita10.JPG"/></a>
+                <a href="images/Nikita11.JPG"><img src="images/Nikita11.JPG"/></a>
+                <a href="images/Nikita12.JPG"><img src="images/Nikita12.JPG"/></a>
```

Comment:
Hmm.. Maybe there are some CSS templates that automatically arrange these into a table so you don't have to hard-code your columns and rows. Especially because these may render differently on different screens

Reply:
+1, swetha it would be helpful if you could link an example? if you did this in your own code that would be a good example :)

Reply:
done